lib/Mojolicious/Plugin/AssetPack.pm

```diff
@@ -82,6 +83,7 @@ sub register {
   $self->out_dir($self->_build_out_dir($config, $app));
   $self->base_url($config->{base_url}) if $config->{base_url};
   $self->_reloader($app, $config->{reloader}) if $config->{reloader};
+  $self->{static_paths} = $app->static->paths;
```

Comment:
This is no good. ```$app->static->paths``` can be changed between loading AssetPack and defining assets, so we can't cache the value here.